src/UserManager.ts

```diff
@@ -563,50 +565,36 @@ export class UserManager {
         return signoutResponse;
     }
 
-    public async revokeAccessToken(): Promise<void> {
+    public async revokeTokens(types?: RevokeTokensArgs): Promise<void> {
```

Comment:
cosmetics: we could name types here `args` like in all other public functions

Reply:
To me `Args` implies a list of named params, I think I might prefer renaming the type to `RevokeTokensTypes` if that makes sense